src/main/java/com/epam/ta/reportportal/core/ElementsCounterService.java

```diff
@@ -42,33 +45,41 @@ public ElementsCounterService(TestItemRepository testItemRepository, LogReposito
 	}
 
 	public Long countNumberOfLaunchElements(Long launchId) {
-		long resultedNumber = 1L;
+		final AtomicLong resultedNumber = new AtomicLong(1L);
 		final List<Long> testItemIdsByLaunchId = testItemRepository.findIdsByLaunchId(launchId);
-		resultedNumber += testItemIdsByLaunchId.size();
-		resultedNumber += logRepository.countLogsByTestItemItemIdIn(testItemIdsByLaunchId);
-		resultedNumber += logRepository.countLogsByLaunchId(launchId);
-		return resultedNumber;
+		resultedNumber.addAndGet(testItemIdsByLaunchId.size());
+		Lists.partition(testItemIdsByLaunchId, BATCH_SIZE).forEach(batch -> {
+			resultedNumber.addAndGet(logRepository.countLogsByTestItemItemIdIn(testItemIdsByLaunchId));
+			resultedNumber.addAndGet(logRepository.countLogsByLaunchId(launchId));
```

Comment:
May be we should add logs for launchId only one time?

Reply:
There are two different types of logs.
Launch logs that applied to launch object
And test item logs that applied to test items inside that launch
So, we need to count both of them